src/hardware/motor.cpp

```diff
@@ -471,15 +471,15 @@ void StepperMotor::setCoilB(COIL_STATE desiredState, uint16_t current) {
     }
 
     // Update the output pin with the correct current
-    analogSet(&PWMCurrentPinInfo[B], currentToPWM(current));
+    analogSet(&PWMCurrentPinInfoB, currentToPWM(current));
 }
 
 
 // Calculates the current of each of the coils (with mapping)(current in mA)
-uint32_t StepperMotor::currentToPWM(uint16_t current) const {
+uint16_t StepperMotor::currentToPWM(uint16_t current) const {
```

Comment:
This number has to be a uint32_t for the PWM set function. The CPU will force a conversion that is not needed